src/protocol/inlineCompletions.ts

```diff
@@ -2,9 +2,9 @@ import {
     InlineCompletionItem,
     InlineCompletionList,
     InlineCompletionParams,
-    InlineCompletionRegistrationOptions,
     ProtocolRequestType,
-} from 'vscode-languageserver'
+    InlineCompletionRegistrationOptions,
+} from './lsp'
```

Comment:
Is there still a copy/paste here from the 3.18 branch of `vscode-languageserver`?

Reply:
yes, 3.18 LSP is not yet published